fragments/infra-boot.sh

```diff
@@ -62,7 +62,7 @@ function use_local_dns() {
 # add the local nameserver to the beginning of the local resolver list
 function add_nameserver() {
     # NAMESERVER_IP=$1
-    sed -i '/search openstacklocal.*/a\nnameserver $1/' /etc/resolv.conf
+    sed -i "/search openstacklocal.*/anameserver $1" /etc/resolv.conf
```

Comment:
this should be squashed with the first patch (which introduced this bug) - it spent a moment when reading the first patch to find out how is it that the broken code worked in deployed stack